packages/orchestration/src/facade.js

```diff
@@ -64,6 +65,8 @@ export const makeOrchestrationFacade = ({
 
   const { prepareEndowment, asyncFlow } = asyncFlowTools;
 
+  const zcfLtd = prepareZcfForFlows(zcf, zone, vowTools);
```

Comment:
I think this needs to be called by `provideOrchestration` using `zones.orchestration` (aka the system orchestration zone), and returned as one of the orchestration tools.

Reply:
What's the impact on contracts?

I started making this change... it's straightforward enough... but it means that all the contracts don't use the attenuated ZCF unless we change them to opt-in. Is that by design?

How does this impact the 1st arg to the `withOrchestration` callback (usually `contract`)? Do we keep the normal zcf there plus the attenuated zcf in the `tools`?

```js
const contract = async (zcf, _privateArgs, zone, tools) => { ... }
export const start = withOrchestration(contract);
```